contexts/AuthContext.tsx

```diff
@@ -109,6 +120,7 @@ export const AuthProvider = ({ children, publicPage = false }: any) => {
         name,
         privateKey,
         publicKey,
+        isVerified,
```

Comment:
C1
After the edit in C0, TS complains because `setUser` requires the `isVerified` key